### PR TITLE
Add python scripts and dataset labels

### PR DESCRIPTION
**Pull request by madhuparnab**

### CODE REVIEW

**Review by sidhantgoyal-droid — COMMENTED** [review]
Proper casing for directory names:
1. dataset_lables
2. helper_scripts

**Comment by madhuparnab** [comment]
> Proper casing for directory names:
> 
> 1. dataset_lables
> 2. helper_scripts

Done.

**Review by sidhantgoyal-droid — DISMISSED** [review]
Unreviewed since there are helper utilities for covid data preparation and model building not related to AI Enrichment Library

File: PythonScripts/ClipPDF.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# importing the required modules 
+import PyPDF2 
+import os
+from os import listdir
+from os.path import isfile, join
+
+#Clip PDFs to size 300kb
+def clipPDF(pdf):
```
Comment (ashish-google) [inline PythonScripts/ClipPDF.py]: Function naming uses underscore in python

Comment (ashish-google) [reply PythonScripts/ClipPDF.py]: Can rename pdf to pdf_file_path?

File: PythonScripts/ClipPDF.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    b = os.path.getsize(pdf)
+    pdfReader = PyPDF2.PdfFileReader(pdfFileObj)
+    pages = pdfReader.numPages 
+    if pages < 3:
+        return
+    end = 0
+    if b < 300000:
+        end = pages
+    else:
+        end = int((pages)*(300000/b))
```
Comment (ashish-google) [inline PythonScripts/ClipPDF.py]: Are all pages of pdf supposed to be of equal size?

**Review by ashish-google — DISMISSED** [review]

File: PythonScripts/ClipPDF.py (hunk 2 above)
Comment (madhuparnab) [reply PythonScripts/ClipPDF.py]: No, but I wasn't able to find the size of each page using the library that I was using. Hence, I used an approximate to decide the number of pages to include.

Comment (madhuparnab) [reply PythonScripts/ClipPDF.py]: Done.

File: PythonScripts/ClipPDF.py (hunk 1 above)
Comment (madhuparnab) [reply PythonScripts/ClipPDF.py]: done

**Review by sidhantgoyal-droid — APPROVED** [review]

**Review by ashish-google — APPROVED** [review]

**Pull request merged**